backend/src/controllers/projects/contacts.js

```diff
@@ -11,19 +12,35 @@ const controller = useController(model, what, "projects");
  * @returns {object}
  */
 controller.updateContacts = async (request, reply) => {
-  let contacts = [];
-  for (const [key, value] of Object.entries(request.body)) {
-    value.map((row) => {
-      contacts.push({
-        contact_role: Number(key),
-        project_id: Number(request.params.id),
-        contact_id: row.value,
-      });
-    });
-  }
-
   try {
-    const result = await model.updateOne(contacts, Number(request.params.id));
+    let contactsFormatted = [];
+    for (const [key, contactsRaw] of Object.entries(request.body)) {
+      if (contactsRaw?.value) {
+        contactsFormatted.push({
+          contact_role: 6,
```

Comment:
if we are putting things that have no meaning to a normal person like 6, then we probably should add a comment to state what 6 means.

Reply:
maybe we could define 6 as a constant, and then put the constant in its place.

Reply:
good comment, added comments ;)